queue_job/job.py

```diff
@@ -645,11 +666,22 @@ def eta(self, value):
         else:
             self._eta = value
 
+    def get_db_txid(self):
```

Comment:
Please add a docstring mentioning that you are not storing the full transaction id but only the 32 bit variant that can be compared with pg_stat_activity's backend_xid